### PR TITLE
feat(monkeys): Create docker image to facilitate execution of the monkeys application [WPB-4558]

### PR DESCRIPTION
**Pull request by augustocdias**
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

This adds a docker stack to execute the monkeys and monitore it. Prometheus and grafana instances will start alongside the monkeys app and automatically scrape metrics from it.

The tests have been marked as `Ignored` because they are flaky inside docker. Once we figure out the reason we'll add them back to the CI.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by typfel** [comment]
>The tests have been marked as Ignored because they are flaky inside docker. Once we figure out the reason we'll add them back to the CI.

But why are we are the tests inside docker on the CI?

**Comment by augustocdias** [comment]
> > The tests have been marked as Ignored because they are flaky inside docker. Once we figure out the reason we'll add them back to the CI.
> 
> But why are we are the tests inside docker on the CI?

They are always inside docker. That's how it runs

File: monkeys/docker/config/config.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
```
Comment (vitorhugods) [inline monkeys/docker/config/config.json:1]: Should this file be here? Doesn't it contain sensitive data?

Comment (augustocdias) [reply monkeys/docker/config/config.json:1]: > Should this file be here? Doesn't it contain sensitive data?

You're right! :|

**Review by typfel — APPROVED** [review]

**Review by vitorhugods — APPROVED** [review]

**Pull request merged**